MNE/disp3D/3DObjects/brain/brainsurfacetreeitem.cpp

```diff
@@ -156,90 +156,90 @@ bool BrainSurfaceTreeItem::addData(const Surface& tSurface, Qt3DCore::QEntity* p
     QVariant data;
 
     data.setValue(arrayCurvatureColor);
-    this->setData(data, BrainSurfaceTreeItemRoles::SurfaceCurrentColorVert);
-    this->setData(data, BrainSurfaceTreeItemRoles::SurfaceCurvatureColorVert);
+    this->setData(data, Data3DTreeModelItemRoles::SurfaceCurrentColorVert);
+    this->setData(data, Data3DTreeModelItemRoles::SurfaceCurvatureColorVert);
 
     data.setValue(tSurface.rr());
-    this->setData(data, BrainSurfaceTreeItemRoles::SurfaceVert);
+    this->setData(data, Data3DTreeModelItemRoles::SurfaceVert);
 
     data.setValue(tSurface.tris());
-    this->setData(data, BrainSurfaceTreeItemRoles::SurfaceTris);
+    this->setData(data, Data3DTreeModelItemRoles::SurfaceTris);
 
     data.setValue(tSurface.nn());
-    this->setData(data, BrainSurfaceTreeItemRoles::SurfaceNorm);
+    this->setData(data, Data3DTreeModelItemRoles::SurfaceNorm);
 
     data.setValue(tSurface.curv());
-    this->setData(data, BrainSurfaceTreeItemRoles::SurfaceCurv);
+    this->setData(data, Data3DTreeModelItemRoles::SurfaceCurv);
 
     data.setValue(tSurface.offset());
-    this->setData(data, BrainSurfaceTreeItemRoles::SurfaceOffset);
+    this->setData(data, Data3DTreeModelItemRoles::SurfaceOffset);
 
     data.setValue(m_pRenderable3DEntity);
-    this->setData(data, BrainSurfaceTreeItemRoles::SurfaceRenderable3DEntity);
+    this->setData(data, Data3DTreeModelItemRoles::SurfaceRenderable3DEntity);
 
     data.setValue(m_pRenderable3DEntityActivationOverlay);
-    this->setData(data, BrainSurfaceTreeItemRoles::SurfaceRenderable3DEntityAcivationOverlay);
+    this->setData(data, Data3DTreeModelItemRoles::SurfaceRenderable3DEntityAcivationOverlay);
 
     //Add surface meta information as item children
     QList<QStandardItem*> list;
 
-    m_pItemSurfColSulci = new BrainTreeMetaItem(BrainTreeMetaItemTypes::SurfaceColorSulci, "Sulci color");
-    connect(m_pItemSurfColSulci, &BrainTreeMetaItem::curvColorsChanged,
+    m_pItemSurfColSulci = new MetaTreeItem(MetaTreeItemTypes::SurfaceColorSulci, "Sulci color");
+    connect(m_pItemSurfColSulci, &MetaTreeItem::curvColorsChanged,
             this, &BrainSurfaceTreeItem::onColorInfoOriginOrCurvColorChanged);
     list<<m_pItemSurfColSulci;
```

Comment:
you always did not put spaces here: but for better readability i would prefer list << m_p...; otherwise it looks more like one word